Cargo.toml

```diff
@@ -37,16 +37,16 @@ serde_derive = "1.0"
 approx = ">=0.3, <0.6"
 custom_derive = "0.1"
 newtype_derive = "0.1"
-ordered-float = "1.0"
+ordered-float = "2.0"
 regex = { version = "1.3", default-features = false, features = ["std", "perf"] }
 multimap = ">=0.6, <0.9"
 fxhash = "0.2"
 statrs = ">= 0.11, < 0.16"
 bio-types = ">=0.11.0"
 pest = { version = "2", optional = true }
 pest_derive = { version = "2", optional = true }
-strum = ">= 0.16, < 0.24"
-strum_macros = ">= 0.16, < 0.24"
+strum = "0.24"
+strum_macros = "0.24"
```

Comment:
Is it not possible to retain this as a version range?  Was there a relevent breaking change between 0.23 and 0.24?  It's much more convenient for downstream users if they don't have to synchronize updates like this.

Reply:
I agree. That was the intention with the range here.

Reply:
```suggestion
strum = ">= 0.16, < 0.25"
strum_macros = ">= 0.16, < 0.25"
```